packages/odyssey-react/.storybook/main.js

```diff
@@ -59,7 +59,6 @@ module.exports = {
                 @import '@okta/odyssey/src/scss/components/input-field';
                 @import '@okta/odyssey/src/scss/components/input-field-layout';
                 @import '@okta/odyssey/src/scss/components/label';
-                @import '@okta/odyssey/src/scss/components/modal';
```

Comment:
`_forms-context.scss` references modal class names here too. Do you believe those are also safe to be removed now?

I think the long term strategy here will be a form `variant="compact"` or similar that can be used within parent Form components cc @edburyenegren-okta

Reply:
I think we can safely drop `forms-context` here. @joejezowski-okta and I will add those styles to the new Form module if necessary.

Reply:
Removed.